src/main/resources/application.yaml

```diff
@@ -140,6 +140,7 @@ application:
   veilarboppfolging:
     api:
       url: ${VEILARBOPPFOLGINGAPI_URL}
+      scope: api://${APP_CLUSTER_NAME}.pto.veilarboppfolging/.default
```

Comment:
Note to self: Vurder å sette denne eksplisitt i nais-yaml, er ikke nødvendigvis clusteret man kjører i som er riktig